src/util/psbt/map/input.rs

```diff
@@ -192,21 +192,23 @@ impl PsbtSigHashType {
 
 impl Input {
     /// Obtains the [`EcdsaSigHashType`] for this input if one is specified.
-    /// If no sighash type is specified, returns ['EcdsaSigHashType::All']
```

Comment:
IMO it was useful to have this as summary but a shorter wording may be nice

Reply:
Cool, thanks, will re-spin after @rish-singhal has had a chance to comment.